nalej/web
# Typing the app

#### What does this PR do?
This pr includes and use new types in the project, the scope has been avoid _**any type**_ in the controller layer, however there are some changes in the service layer. Last but not least, we have a new rule in the tslint file to avoid new _**any type**_ in the code.

#### Where should the reviewer start?
- src/app/definitions/**

#### What is missing?
I had to reduce the scope because the task was coming infinite, so there are some // tslint:disable:no-any in the project, we'll delete gradually them. 

#### How should this be manually tested?
1. npm start
2. Navigate across the app and verify the most important operations related to deployment, graph painting, labels interaction, instance view info, registered info, user management, devices management, etc
 
#### What are the relevant tickets?
- [NP-2067](https://nalej.atlassian.net/browse/NP-2067)
